hazelcast-quarkus-client.yml

```diff
@@ -13,7 +13,7 @@ services:
     image: mesut/hazelcast-quarkus-native
     environment:
       - CONTAINER_NAME=hazelcast-quarkus_2
-      - HAZELCAST_IP=node1
+      - HAZELCAST_IP=node2
```

Comment:
As discussed privately, it would be better explain this in the README. Maybe with a diagram to show... In the hazelcast-client, you can define one member (node1) in the whole example and it will fetch IP adresseses of the others. Then, each quarkus microservice will connect to all hazelcast members(smart routing feature).

Reply:
I also think we should rename thing to hazelcast_member_1, hazelcast_member_2... node is not the right terminology. I made a mistake to use it initially.